binance_trade_bot/binance_api_manager.py

```diff
@@ -32,6 +37,9 @@ def get_using_bnb_for_fees(self):
         return self.binance_client.get_bnb_burn_spot_margin()
 
     def get_fee(self, origin_coin: Coin, target_coin: Coin, selling: bool):
+        if self.TRADE_FEE:
```

Comment:
Why not do 
```suggestion
        if self.config.TRADE_FEE == 'auto':
            ...
        else:
            return float(self.config.TRADE_FEE)
```

We don't have any other configuration in the `__init__` method and I think we should keep it that way

Reply:
true, yet binance_api_manager doesn't have config itself. Need to use self.binance_client.config.TRADE_FEE

Reply:
True, yet binance_api_manager doesn't store config

Reply:
I'd make it store the config then. I know another PR has also done that.